python/triton/runtime/autotuner.py

```diff
@@ -261,10 +272,13 @@ def kernel(x_ptr, x_size, **META):
     :type warmup: int
     :param rep: Repetition time (in ms) to pass to benchmarking, defaults to 100.
     :type rep: int
+    :param report: Flag to enable printing the selected configuration
```

Comment:
Rename the flag to `print`?  "report" can be a noun or a verb, and that ambiguity is confusing here.  (Do I pass the report in as a parameter?  What is the report?  Or is the autotuning report passed as an outparameter somehow?)

Reply:
ok, changed it to `print_autotune_stats`